### PR TITLE
Gra 1191 legacy nodes

### PR DESCRIPTION
**Pull request by mattkasun**
updated migration function

### CODE REVIEW

**Review by alphadose — APPROVED** [review]

**Review by Aceix — APPROVED** [review]

**Review by theguy951357 — APPROVED** [review]
pass

**Review by theguy951357 — COMMENTED** [review]
changes needed are noted in GRA-1191

**Comment by theguy951357** [comment]
testing fails on windows.
testing notes and log and config files in gra-1191 ticket.

File: controllers/migrate.go [hunk 1]
```diff
@@ -74,5 +75,31 @@ func migrate(w http.ResponseWriter, r *http.Request) {
 	}
 	r.Body = io.NopCloser(strings.NewReader(string(payload)))
 	r.ContentLength = int64(len(string(payload)))
+	pretty.Println(data.JoinData)
```
Comment (0xdcarns) [inline controllers/migrate.go:78]: pretty print needed?

**Review by 0xdcarns — CHANGES_REQUESTED** [review]

**Review by theguy951357 — APPROVED** [review]
pr passes pending changes addressed.

File: controllers/migrate.go (hunk 1 above)
Comment (mattkasun) [reply controllers/migrate.go]: removed debug log

**Review by 0xdcarns — APPROVED** [review]

**Pull request merged**